applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h

```diff
@@ -321,26 +291,20 @@ namespace Kratos {
 
                     if (mMeshVelocity == 2)//Lagrangian
                     {
-                        array_1d<double, 3 > & CurrentDisplacement = (itNode)->FastGetSolutionStepValue(DISPLACEMENT, 0);
-                        array_1d<double, 3 > & OldDisplacement = (itNode)->FastGetSolutionStepValue(DISPLACEMENT, 1);
-
-                        array_1d<double, 3 > & OldVelocity = (itNode)->FastGetSolutionStepValue(VELOCITY, 1);
-
-    
-                        
                         if((itNode)->FastGetSolutionStepValue(IS_LAGRANGIAN_INLET) < 1e-15)
```

Comment:
What is this if-statement checking? If you wrap this in a inline function with a descriptive name it would be much more understandable.

Reply:
let me redirect this question to @RiccardoRossi, I've never acutally used this scheme in "lagrangian" mode... If I had to guess, however, this is/was used to detect nodes that belong to parts of the mesh that should not be moved even if the corresponding nodes have a non-zero velocity. @RiccardoRossi is this still the recommended way to check for this, or is there a better flag to use?